scapy/automaton.py

```diff
@@ -836,8 +837,7 @@ def __init__(self, *args, **kargs):
         self.init_kargs = kargs
         self.io = type.__new__(type, "IOnamespace", (), {})
         self.oi = type.__new__(type, "IOnamespace", (), {})
-        self.cmdin = ObjectPipe[Message]("cmdin")
-        self.cmdout = ObjectPipe[Message]("cmdout")
+        self.cmdin = self.cmdout = None  # type: ObjectPipe[Message]  # type: ignore
```

Comment:
Would this work?
```suggestion
        self.cmdin = self.cmdout = None  # type: Optional[ObjectPipe[Message]]
```

Reply:
I'm actually cheating here because `cmdin` and `cmdout` are always !=None when the automaton is started (after `.start()`, i.e. on initialization) and it would be a pain to add a `if <...> is not None:` everywhere. We want to have that in `start()` rather than `__init__` to handle the special `restart()` case where they'll be closed then re-opened

Reply:
OK understood. Do we **have** to set them in `__init__()` then?

Reply:
No ! you're right we can just omit it